heka/files/heka_logrotate.conf

```diff
@@ -0,0 +1,19 @@
+/var/log/{{ service_name }}.log {
+  # Heka cannot be told to close its log file and re-open it so we need to
+  # use the copytruncate option
+  copytruncate
+  compress
+  delaycompress
+  missingok
+  notifempty
+  # logrotate allows to use only year, month, day and unix epoch
+  dateext
+  dateformat -%Y%m%d-%s
+  # number of rotated files to keep
+  rotate 10
+  # do not rotate files unless both size and time conditions are met
+  hourly
```

Comment:
'hourly' is useless b/c logrotate is executed daily basis.
If we absolutely need  to rotate hourly we must run a dedicated crontab for that (and don't put this conf in standard logrotate conf directory).
see https://github.com/openstack/fuel-plugin-lma-collector/blob/bb67a130628477de4e3d23e9490ed854e7fd959f/deployment_scripts/puppet/modules/heka/manifests/init.pp#L224-L252

and 
https://github.com/openstack/fuel-plugin-lma-collector/blob/bb67a130628477de4e3d23e9490ed854e7fd959f/deployment_scripts/puppet/modules/heka/templates/logrotate.cron.erb#L33